src/com/TTT/PlayerMarkers.java

```diff
@@ -1,16 +1,22 @@
 package com.TTT;
 
 public enum PlayerMarkers {
-    X ("\u001B[31m " + "X" + " \u001B[0m"),
-    O("\u001B[34m " + "O" + " \u001B[0m");
+    X ("X", "\u001B[31m  X \u001B[0m"),
+    O ("O", "\u001B[34m  O \u001B[0m");
 
```

Comment:
Looking at this, an enum may not have been the way to go. I played about with adding an empty space to this and it doesn't fit with the rest of it.